custom_components/mqtt_vacuum_camera/__init__.py

```
@@ -72,6 +73,18 @@
         # Optionally, trigger other reinitialization steps if needed
         hass.bus.async_fire(f"event_{DOMAIN}_reloaded", context=call.context)
 
+
+
+    async def vacuum_goto(call: ServiceCall) -> None:
+        """Vacuum Go To Action"""
+        entity_id = call.data["entity_id"]
+        x = call.data["x"]
```

Ruff

Local variable `x` is assigned to but never used

Remove assignment to unused variable `x`

(F841)